proguard.txt

```diff
@@ -1,10 +1,11 @@
--dontobfuscate
+-keeppackagenames com.bugsnag.**
+-keepnames class com.bugsnag.**
+-keepclassmembernames class com.bugsnag.** { *; }
 
--renamesourcefileattribute SourceFile
 -keepattributes Exceptions,InnerClasses,Signature,Deprecated,
                 SourceFile,LineNumberTable,*Annotation*,EnclosingMethod
 
--keep public class * {
+-keep public class com.bugsnag.** {
```

Comment:
In the current master the proguard rules are only used during the build of the library itself so only classes in this library are affected by `-keep public class *`.

With this PR the proguard rules aren't used during the build of library anymore but during the build of the app which includes the library. With the change above we make sure the rule still only affects the library and not the whole app